h2o-algos/src/main/java/hex/DataInfo.java

```diff
@@ -521,7 +521,7 @@ public DataInfo filterExpandedColumns(int [] cols){
       ignoredCols[ignoredCnt++] = k+_cats;
     Frame f = new Frame(_adaptedFrame.names().clone(),_adaptedFrame.vecs().clone());
     if(ignoredCnt > 0) f.remove(Arrays.copyOf(ignoredCols,ignoredCnt));
-    assert catLvls.length < f.numCols():"cats = " + catLvls.length + " numcols = " + f.numCols();
+    assert catLvls.length <= f.numCols():"cats = " + catLvls.length + " numcols = " + f.numCols();
```

Comment:
This makes sense but it would be good to add a small unit test to check it.

Reply:
There is the test case (the second one) which test it in runit_pubdev_4659.R. It is the case where there is only one column of data and no response column in test data (no weights, offset column, etc.).